script.js

```diff
@@ -0,0 +1,205 @@
+               //outputModalContent.innerHTML += ('round: '+rou+'/'+pla+'/'+com+'/'+win+'/'+statu+'/'+'<br>');                               
+           }  
+           document.querySelector('#modal-overlay').classList.add('show');
+           document.querySelector('#modal-one').classList.add('show');
+           document.querySelector('#modal-two').classList.remove('show');           
+           
+        } 
+        var table2 = function() {
+            for (var i = 0; i < params.progress.length; i++) {
+                var rou = params.progress[i].round;
+                var pla = params.progress[i].player;
+                var com = params.progress[i].computer;
+                var win = params.progress[i].winnerIs;
+                var statu = params.progress[i].roundStatus;
+                outputModalContent2.innerHTML += '<span style="margin: 0 auto">'+'<table style="border: 1px solid black;">' + '<tr>' + '<td>' + rou + '</td>' + '<td>' + pla + '</td>' + '<td>' + com + '</td>'+ '<td>' + win + '</td>'+ '<td>' + statu + '</td>' + '</tr>' + '</table>'+'</span>';
+            }  
+            document.querySelector('#modal-overlay').classList.add('show');
+            document.querySelector('#modal-two').classList.add('show');
+            document.querySelector('#modal-one').classList.remove('show');
+        }       
+
+        function handleClick(choice) {
+            if (params.userWinRate < params.levelCap && params.computerWinRate < params.levelCap ){                                      
+                palyerMove(choice);                
+                outputPlayer.innerHTML = params.userWinRate;
+                outputComputer.innerHTML = params.computerWinRate;
+                outputRoundsPlayed.innerHTML = '<h2> Move count: '+params.roundsPlayed+'</h2>';
+           }      
+           else if (params.userWinRate == params.levelCap && params.computerWinRate < params.levelCap) { 
+                    outputModalContent.innerHTML += '<span style="margin: 0 auto; font-weight: bold;">'+'<table style="border: 1px solid black;">' + '<tr>' + '<td>' + 'round' + '</td>' + '<td>' + 'player move' + '</td>' + '<td>' + 'computer move' + '</td>'+ '<td>' + 'win/lost' + '</td>'+ '<td>' + 'game status' + '</td>' + '</tr>' + '</table>'+'</span>';                       
+                    table1();
+                    //stopPropagation;
+           }
+           else if (params.computerWinRate == params.levelCap && params.userWinRate < params.levelCap) { 
+                    outputModalContent2.innerHTML += '<span style="margin: 0 auto; font-weight: bold;">'+'<table style="border: 1px solid black;">' + '<tr>' + '<td>' + 'round' + '</td>' + '<td>' + 'player move' + '</td>' + '<td>' + 'computer move' + '</td>'+ '<td>' + 'win/lost' + '</td>'+ '<td>' + 'game status' + '</td>' + '</tr>' + '</table>'+'</span>';
```

Comment:
jak będzie potrzeba zmienić np `font-weight` to będziesz to musiał zrobić w kilku miejscach